src/pages/index.js

```diff
@@ -83,20 +71,65 @@ const editProfileFormValidator = new FormValidator(
   settings,
   editProfileModalFormElement
 );
+const editAvatarFormValidator = new FormValidator(
+  settings,
+  editAvatarModalFormElement
+);
+const api = new Api({
+  baseUrl: "https://around.nomoreparties.co/v1/cohort-3-en",
+  headers: {
+    authorization: "b685d3e0-616a-4dae-bc5b-53892a4f7953",
+    "Content-Type": "application/json",
+  },
+});
+function createCard(item) {
+  // create instance of Card class
+  const card = new Card(
+    item,
+    "#card-template",
+    onCardClick,
+    handleCardDeleteFunctionInIndexComponent
+  );
+  //create a card by calling getCardElement method from Card class
+  const cardElement = card.getCardElement();
+  //return the card
+  return cardElement;
+}
+
+api.getUserInfo().then((data) => {
+  const name = data.name;
+  const subtitle = data.about;
+  const url = data.avatar;
+  userInfo.setUserInfo({ name, subtitle });
+  userInfo.setNewAvatar(url);
+});
+
+api
+  .getInitialCards()
+  .then((data) => {
+    // process the result
+    data.forEach((item) => {
+      const cardItem = createCard(item);
+      section.appendItem(cardItem);
+    });
+  })
+  .catch((err) => {
+    console.error(err); // log the error to the console
+  });
 
-// To render initial cards declared above in cardData array
 const section = new Section(
-  { items: cardData, renderer: createCard },
+  { items: api.getInitialCards, renderer: createCard },
   cardsList
 );
-section.renderItems();
 
 function handleOpenEditProfileForm() {
   editProfileFormValidator.disableButton();
+  api.getUserInfo().then((result) => {
```

Comment:
No need to call `api.getUserInfo()` when you open the profile modal because it's a waste of time and server resources. You have method `getUserInfo` inside class `UserInfo` for that